apps/network-assembler/networkassembler.yml

```diff
@@ -0,0 +1,12 @@
+name: NetworkAssembler
```

Comment:
Please rename this file to `network-assembler.yml`, and icon to `network-assembler-icon.png`